src/components/HomePage/UpcomingEvents.js

```diff
@@ -9,9 +9,11 @@ import { Link } from 'gatsby';
 import eventSchedule from '../../data/eventSchedule';
 import Event from './Event';
 import NoSsr from '@material-ui/core/NoSsr';
+import { hothStart } from '../constants';
 
 // Schedule will be available on the homepage the day before HOTH
-const scheduleAvailableTime = new Date('2022-02-25T00:00:00-08:00');
+const dayBefore = 86400000; // 24 hours to milliseconds
+const scheduleAvailableTime = new Date(hothStart - dayBefore);
```

Comment:
Actually, can you just move `scheduleAvailableTime` into `constants.js`? 
Potentially it could be used by other files as well + for a better separation of concerns.